c2corg_ui/static/partials/map/map.html

```diff
@@ -8,7 +8,11 @@
     class="btn btn-primary map-reset-feature" translate>Reset feature</button>
   <div class="map-filter-switch" ng-show="::mapCtrl.advancedSearch">
     <label>
-      <input type="checkbox" ng-model="mapCtrl.enableMapFilter"><span translate>Filter on map extent</span>
+      <input type="checkbox" ng-model="mapCtrl.enableMapFilter">
+      <div>
+        <span translate class="text">Filter on map extent</span>
+        <span class="glyphicon glyphicon-globe"></span>
```

Comment:
I get it: the globe is shown only on mobile screen, instead of the text. That looks like a good idea indeed, even though showing only the icon is not very explicit.